src/tenant/service/tenant.service.interface.ts

```diff
@@ -0,0 +1,10 @@
+import { NewTenantInput } from 'src/schema/graphql.schema';
+import Tenant from '../entity/tenant.entity';
+
+export interface TenantServiceInterface {
+  getTenantByDomain(domain: string): Promise<Tenant>;
```

Comment:
Yes, this function was planned to be part of the login procedure for multitenancy. However, the implementation was put on hold as the approach for multitenancy login wasn't confirmed, and it will be part of a future commit.